README.md

```diff
@@ -70,13 +70,13 @@ Mode 2 can be used to verify the Suyama A residue stored in a verifiable delay f
 ```bash
 cofact -cpr F14.proof 14
 ```
-This performs a Pepin primality test for $F_{14}$, then calculates the Suyama A residue and compares it with the residue stored in the proof file. If no residue mismatch is reported, then the proof file has a valid residue. You may check any proof file with or without specifying factors.
+This performs a Pépin primality test for $F_{14}$, then calculates the Suyama A residue and compares it with the residue stored in the proof file. If no residue mismatch is reported, then the proof file has a valid residue. You may check any proof file with or without specifying factors.
 
 Mode 3 can be used to test a Fermat cofactor using the Suyama A residue stored in the corresponding proof file. For example, to test the $F_{14}$ cofactor using the $F_{14}$ proof file, type:
 ```bash
 cofact -upr F14.proof 14 116928085873074369829035993834596371340386703423373313
 ```
-Mode 3 avoids the lengthy Pepin calculation, allowing a cofactor to be tested in a matter of minutes even for Fermat numbers as large as $F_{29}$. To enable yourself to test the resulting cofactor after the next factor of $F_{14}$ through $F_{29}$ is discovered, download the $F_{14}$ through $F_{29}$ proof files from Catherine's excellent [website](https://64ordle.au/fermat/).
+Mode 3 avoids the lengthy Pépin calculation, allowing a cofactor to be tested in a matter of minutes even for Fermat numbers as large as $F_{29}$. To enable yourself to test the resulting cofactor after the next factor of $F_{14}$ through $F_{29}$ is discovered, download the $F_{14}$ through $F_{29}$ proof files from Catherine's excellent [website](https://64ordle.au/fermat/).
 
 The cofact distribution includes a script called `run_all` that will run cofact on each Fermat number from $F_0$ through $F_{29}$ using the best method for that number, with "best" meaning reasonably fast. Before running the script, download proof files for $F_{17}$ through $F_{29}$ into the same directory as cofact.
```

Comment:
To compensate for removing the somewhat similar content from line 79, I would amend this line as follows,

The cofact distribution includes a script called `run_all` that will run cofact on each Fermat number from $F_0$ through $F_{29}$ using the best method for that number, with "best" meaning reasonably fast. Before running the script, you should have proof files for $F_{17}$ through $F_{29}$ in the same directory as cofact. A complete set of proofs for $F_{12}$ through $F_{29}$ is available from Catherine's [website](https://64ordle.au/fermat/) which will allow cofactor tests whenever a new factor of one of these Fermat numbers is discovered.